dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -49,6 +52,7 @@
 
     private static final String CASE_FILE = "case-file";
     private static final String DYNAMIC_MODEL_FILE = "dynamic-models-file";
```

Comment:
@marcosmc could you also rename the variable DYNAMIC_MODEL_FILE into DYNAMIC_MODELS_FILE? I forgot to do it in my PR. Thanks.

Reply:
Done